Dockerfile

```diff
@@ -1,16 +1,27 @@
 # ============ Build Stage ============
-FROM rust:1.74-bookworm as build
-
+FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
```

Comment:
For reproducibility, I recommend adding the digest hash like `@sha256:...` after the version name (`docker inspect <image> | grep @sha`).

Reply:
I think I will just change the tag to something not dynamic

Reply:
The digest is the only non-dynamic thing. Any tag you use can be updated to a different base image in the upstream.

Reply:
You've convinced me to continue using the latest stable Rust. Rust never has breaking syntax changes. Cargo-chef, which is more likely to introduce incompatible changes, also publishes bug-fixes, so I'll also keep it latest. 
I'll try to switch to a slim build image. 
I think switching to Alpine-based runtime is out of scope right now. A significant part of the size is the fonts.

Reply:
I like reproducibility, and I would prefer for the final image to be as small as possible. But it's up to you how to achieve that and what's feasible.

Reply:
Testing `latest-rust-slim` creates an error. Sourcing out to a new issue and resolving.